docs/guides/creating-a-smb-share.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Sharing a Directory
+nav_exclude: false
+nav_order: false
+has_children: false
+parent: Guides
+search_exclude: false
+last_modified_date: 2023-02-03
+---
+
+# Sharing a Directory on Your Local Network
+
+## Permissions
+
+Before sharing a directory, it's important to ensure that the correct permissions are set. To share a directory, the user account that you use must have read and write permissions for that directory. To set permissions, follow these steps:
+
+1. Right-click the directory that you want to share.
+2. Click Properties.
+3. Click the Sharing tab.
+4. Click Advanced Sharing.
+5. Click Permissions.
+6. Add the user accounts that you want to share with, and set their permissions as desired.
+
+## Using Microsoft Accounts
+
+If you are using Microsoft accounts, you can share a directory with other Microsoft account users over your local network. To do this, follow these steps:
+
+1. Right-click the directory that you want to share.
+2. Click Properties.
+3. Click the Sharing tab.
+4. Click Advanced Sharing.
+5. Check the box for "Share this folder".
+6. Enter a name for the share.
+7. Click Permissions.
+8. Add the Microsoft account users that you want to share with, and set their permissions as desired.
+
+## The Network Centere

```

Comment:
you can't choose both the american and british spelling choose one
`## The Network Center`
`## The Network Centre`